ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -661,11 +661,11 @@ public class KsqlConfig extends AbstractConfig {
   private static final boolean KSQL_ENDPOINT_MIGRATE_QUERY_DEFAULT = true;
   private static final String KSQL_ENDPOINT_MIGRATE_QUERY_DOC
       = "Migrates the /query endpoint to use the same handler as /query-stream.";
-  public static final String KSQL_ASSERT_TOPIC_DEFAULT_TIMEOUT_MS
-      = "ksql.assert.topic.default.timeout.ms";
-  private static final int KSQL_ASSERT_TOPIC_DEFAULT_TIMEOUT_MS_DEFAULT = 1000;
-  private static final String KSQL_ASSERT_TOPIC_DEFAULT_TIMEOUT_MS_DOC
-      = "The default timeout for ASSERT TOPIC statements if not timeout is specified "
+  public static final String KSQL_ASSERT_DEFAULT_TIMEOUT_MS
```

Comment:
For those playing the home game, this introduced in https://github.com/confluentinc/ksql/commit/cd5254f9214fe0faadb98d505664f866805aedf0 which has not been released, so nothing public and published is changing.  (I had to check for my own sanity.:))